src/ucp/wireup/select.c

```diff
@@ -1440,6 +1481,11 @@ ucp_wireup_construct_lanes(const ucp_wireup_select_params_t *select_params,
         key->lanes[lane].dst_md_index = select_ctx->lane_descs[lane].dst_md_index;
         addr_indices[lane]            = select_ctx->lane_descs[lane].addr_index;
 
+        if (select_ctx->lane_descs[lane].usage & UCP_WIREUP_LANE_USAGE_CM) {
+            ucs_assert((key->cm_lane == UCP_NULL_LANE) ||
+                       (key->cm_lane == lane));
```

Comment:
when is this case possible? (key->cm_lane == lane)

Reply:
when it has been initialized, just to make sure that the value stays the same

Reply:
just select CM lane directly like done for wireup lane in lines 1525-1526